src/styles/stars.scss

```diff
@@ -0,0 +1,25 @@
+.stars {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    background-image: url(../images/star.svg);
+    margin-right: 4px;
+    background-position: center;
+    background-repeat: no-repeat;
+  }
+
+  &__star:nth-child(-n + 4) {
+    background-image: url(../images/star-active.svg);
+  }
+
+  &__text {
+    font-family: $font;
+    font-size: 10px;
+    line-height: 14px;
+    font-weight: $fontWeightStandard;
```

Comment:
Ensure that the variable $font and $fontWeightStandard are defined in 'variables.scss' and imported into this file to maintain consistency and reusability across the project.